components/ErrorModal.vue

```diff
@@ -32,18 +32,31 @@
         class="block not-first:mt-4 px-2 py-1 text-xs font-mono font-medium rounded-md border border-gray-300 bg-gray-100 break-all whitespace-pre-wrap"
         v-text="rawMessage"
       />
+
+      <div
+        v-if="props.actions?.length"
+        class="flex justify-end gap-2 pt-4"
+      >
+        <UButton
+          v-for="(action, index) in props.actions"
+          :key="index"
+          :label="action.label"
+          color="neutral"
+          @click="action.click"
+        />
+      </div>
     </template>
   </UModal>
 </template>
 
 <script setup lang="ts">
 const emit = defineEmits(['close'])
 
-const props = defineProps({
-  title: String,
-  description: String,
-  rawMessage: String,
-})
-
+const props = defineProps<{
+  title: string
+  description: string
+  rawMessage: string
+  actions?: Array<{ label: string, click: () => void }>
```

Comment:
Please `import type { ButtonProps } from '@nuxt/ui'`
```suggestion
  actions?: Array<ButtonProps>
```